plugins/discovery-azure/src/test/java/org/elasticsearch/cloud/azure/AbstractAzureComputeServiceTestCase.java

```diff
@@ -48,7 +48,7 @@ protected Settings nodeSettings(int nodeOrdinal) {
 
         // We add a fake subscription_id to start mock compute service
         builder.put(Management.SUBSCRIPTION_ID, "fake")
-            .put(Discovery.REFRESH, "5s")
+            .put(Discovery.REFRESH_SETTING.getKey(), "5s")
```

Comment:
May be I don't understand how the new setting infra works but it looks strange to me.

I'd expect here that `Discovery.REFRESH_SETTING.getKey()` is actually `discovery.azure.refresh_interval`

Do I misunderstand?